docs/content/en/contribute/general/github/_index.md

```diff
@@ -0,0 +1,276 @@
+     * Click the **Fork** button at the top of the screen.
+     * Choose the user for the fork from the options you are given,
+       usually your GitHub ID.
+
+   A copy of this repository is now available in your GitHub account.
+
+2. Get the string to use when cloning your fork:
+
+     * Click the green "Code" button on the UI page.
+     * Select the protocol to use for this clone (either HTTPS or SSH).
+     * A box is displayed that gives the URL for the selected protocol.
+       Click the icon at the right end of that box to copy that URL.
+
+3. Run the **git clone** command from the shell of a local directory
+    to clone the forked repository to a directory on your local machine,
+    pasting in the URl you saved in the previous step:
+
+    ```console
+    git clone https://github.com/<UserName>/lifecycle-toolkit
+    ```
+
+    or
+
+    ```console
+    git clone git@github.com:<UserName>/lifecycle-toolkit.git
+    ```
+
+    Where <*UserName*> is your GitHub username.
+    The lifecycle-toolkit directory is now available in the local directory.
+
+4. Associate your clone with `upstream`.
+   To do this, use the same string you used to clone your fork.
+
+   * Be sure that you are in the root folder of the project
+     and run *git status* to confirm that you are on the `main` branch.
+   * Type the following to associate `upstream` with your clone,
+     pasting in the string for the main repo that you copied above.:
+
+     ```console
+     git remote add upstream https://github.com/keptn/lifecycle-toolkit.git 
+     ```
+
+### Create a new branch and make your changes
+
+Now you have your upstream setup in your local machine.
+You need to create a local branch where you will make your changes.
+Be sure that your branch is based on and sync'ed with `main`,
+unless you intend to create a derivative PR:
+
+The following sequence of commands does that:
+
+```console
+git checkout main
+git pull upstream main
+git push origin main
+git checkout -b <my-new-branch>
+```
+
+Now you can make your changes, build them locally to check formatting,
+then create a PR to add these changes to the documentation set
+
+### Submitting new content through a pull request
+
+If you have forked and cloned the repository,
+you can modify the documentation or add new material
+by editing the markdown file(s) in the local clone of your fork
+and then submitting a *pull request (PR)*.
+
+Note that you can also modify the source using the GitHub editor.
+This is very useful when you want to fix a typo or make a small editorial change
+but, if you are doing significant writing,
+it is generally better to work on files in your local clone.
+
+1. Execute the following and check the output to ensure that your branch is set up correctly:
+
+   ```console
+   git status
+   ```
+
+1. Do the writing you want to do in your local branch, checking the formatted version at `localhost:1314`.
```

Comment:
This is only relevant to the docs, not software, and it kind of duplicates what is above only with more detail.  I'm thinking that the section about documentation will describe how to run a local build to test the docs -- maybe better to just reference that in the previous section when we talk about testing your changes before creating the PR.  And not really relevant to this section.